modules/aws/vpc.go

```diff
@@ -143,6 +153,60 @@ func GetSubnetsForVpcE(t testing.TestingT, vpcID string, region string) ([]Subne
 	return subnets, nil
 }
 
+// GetTagsForVpc gets the tags for the specified VPC.
+func GetTagsForVpc(t testing.TestingT, vpcID string, region string) map[string]string {
+	tags, err := GetTagsForVpcE(t, vpcID, region)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I know this whole file doesn't follow this pattern, but we've been incrementally switching to usage of `require` here:
```suggestion
	require.NoError(t, err)
```

Can you update the other new functions to use this pattern too? Thanks!

Import path for `require` is: `"github.com/stretchr/testify/require"`.

Reply:
OK. Updated `GetTagsForVpcE` and `GetTagsForVpc` to make use of this.

![image](https://user-images.githubusercontent.com/28236991/135653947-5453f74a-e6c5-4299-8a7e-545fef96dee7.png)